test/e2e/externalservices/externalservices_multizone_universal.go

```diff
@@ -57,9 +60,12 @@ networking:
 		// External Service non-Kuma Cluster
 		external = clusters.GetCluster(Kuma3)
 
+		// todo(lobkovilya): use test-server as an external service
```

Comment:
Is it a lot of work? Would it be possible to do it now? If now, can we create a card for this?

Reply:
We deploy test-server always with a sidecar, so it's not as easy as just using `TestServerUniversal(...)`. I'd better create a new issue for that